packages/veui/src/components/Uploader.vue

```diff
@@ -150,16 +153,69 @@
               name="file-before"
               v-bind="getScopeValue(index, file)"
             />
-            <div :class="$c('uploader-list-image-container')">
+            <div :class="$c('uploader-list-media-container')">
               <img
+                v-if="getMediaType(file) === 'image'"
                 :src="file.src"
                 :alt="file.alt || ''"
-                :class="$c('uploader-list-image-container-image')"
+                :class="$c('uploader-list-media-container-media')"
+              >
+              <img
+                v-else-if="getMediaType(file) === 'video' && file.poster"
+                :src="file.poster"
+                :alt="file.alt || ''"
+                :class="$c('uploader-list-media-container-media')"
               >
+              <video
+                v-else-if="getMediaType(file) === 'video'"
+                controls
+                :src="file.src"/>
               <div :class="`${listClass}-mask`">
-                <template v-for="control in getImageControls(file)">
+                <template v-for="(control, controlIndex) in getImageControls(file)">
+                  <veui-dropdown
+                    v-if="control.children && control.children.length"
+                    :key="`${control.label}-${controlIndex}`"
+                    :class="$c('control-item')"
+                    trigger="click"
+                    :options="control.children"
+                    @click="handleImageControl(file, index)"
+                    @expandchange="handleDropdownExpand(arguments, index)"
```

Comment:
```suggestion
                    :expanded.sync="expandedControlDropdowns[index]"
```

这里直接双向同步 prop 状态就可以了，handleDropdownExpand 不需要了（名字我也改了一下）